tools/distribution/src/release/assets/gh_asset.py

```diff
@@ -251,6 +251,33 @@ def validate(self, zip_directory: DirectoryMatcher, in_version: Version) -> bool
             ])
 
         return True
+    
+class OpenTelemetryXCFrameworkValidator(XCFrameworkValidator):
+    name = 'OpenTelemetryApi.xcframework'
+
+    def validate(self, zip_directory: DirectoryMatcher, in_version: Version) -> bool:
+        min_otel_version = Version('2.12.0')
```

Comment:
nit: this could be added at the top of the file with the other version requirements (though it's only used by this function for now)

Reply:
This min version only applies to otel, so it can be contained in `OpenTelemetryXCFrameworkValidator`. We can make it global when we need to share it 👍